docs/build.md

```diff
@@ -43,6 +44,7 @@ spec:
 - `source`: The source location that will be the input to the build. See the [Source Configuration](#source-config) section below.
 - `cacheName`: Optional name of a persistent volume claim to used for a build cache across builds.
 - `env`: Optional list of build time environment variables.
+- `projectDescriptorPath`: Path to the [project descriptor file](https://buildpacks.io/docs/reference/config/project-descriptor/) relative to source root dir or `subPath`. If unset, kpack will look for `project.toml` at the root dir.
```

Comment:
Should we clarify what root dir means here? It should look at the subpath project.toml if subpath is specified right?